x-pack/plugins/security_solution/public/resolver/models/process_event.ts

```diff
@@ -32,10 +32,13 @@ export function isTerminatedProcess(passedEvent: ResolverEvent) {
  * ms since unix epoc, based on timestamp.
  * may return NaN if the timestamp wasn't present or was invalid.
  */
-export function datetime(passedEvent: ResolverEvent): number {
+export function datetime(passedEvent: ResolverEvent): number | null {
```

Comment:
@michaelolo24 when the value isn't defined or can't be parsed this now returns `null`.